.github/workflows/check-image-vulnerabilities.yml

```diff
@@ -32,17 +26,51 @@ jobs:
           } >> "$GITHUB_STEP_SUMMARY"
 
   check-image-vulnerabilities:
-    name: Checks images in version ${{ inputs.version }} for vulnerabilities
+    name: Check images in version ${{ inputs.version }} for vulnerabilities
     runs-on: ubuntu-latest
     env:
-      QUAY_BEARER_TOKEN: ${{ secrets.QUAY_RHACS_ENG_VULNERABILITY_BEARER_TOKEN }}
```

Comment:
I think this changes should be in:
https://github.com/stackrox/stackrox/blob/2333f8046ebacb348fe5ca33ee327cb0f5c3f69b/.github/workflows/build.yaml#L645

Reply:
I'm not convinced that roxctl scanning as part of `build` workflow provides enough value. I will look into that next (e.g. maybe we can do weekly scanning), so I see it separate from this PR.

Reply:
`scan-imagex-with-roxctl` build check is currently skipped (though not obvious):
- https://github.com/stackrox/stackrox/blob/2333f8046ebacb348fe5ca33ee327cb0f5c3f69b/release/scripts/scan-images-with-roxctl.sh#L10-L17 --> emits `Image scan with roxctl is disabled`
- Example: https://github.com/stackrox/stackrox/actions/runs/8784472726/job/24103852754#step:10:5

The secret value for `STACKROX_CI_INSTANCE_CENTRAL_HOST` is currently `disabled`, because the `***` in the output above. 

If we were to change this secret, that would affect also this action. 

IMO we should understand why the scan is disabled in build stage, before we take the approach to move this check from release to build (where we can refactor it to a GHA from the shell script to dogfood the central-login and install-roxctl actions as well).

Reply:
cc @gavin-stackrox as the author of the `scan-images-with-roxctl` part.

Reply:
> why the scan is disabled in build stage
This was disabled because the CI instance died.

Reply:
> This was disabled because the CI instance died.

if that is the only reason, let's go straight to running this check on each master & tag push.

Reply:
What would be the purpose of scanning on each master push? Who will read the results and act on them? It seems like something that will just be ignored (rightfully so imo) until it actually matters in the release.

Reply:
> I'm not convinced that roxctl scanning as part of build workflow provides enough value (at least not for every master commit). I will look into that next (e.g. maybe we can do weekly scanning), so I see it separate from this PR.

I misread your comment. Let's merge this first and then add it out to nightlies (they are reviewed by support on-call), and other new tags (release candidates & releases - reviewed by release engineer).